### PR TITLE
Bug OCPBUGS-15370: update Dockerfile.rhel7 with Dockerfile content

### PR DESCRIPTION
**Pull request by jzding**
also update builder from golang-1.19 to golang-1.20 in Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
```
Comment (josephdrichard) [inline Dockerfile:1]: is there a rhel-9 base instead now?

Comment (nishant-parekh) [reply Dockerfile:1]: there is one but I don't think any of the operators have updated them yet.

**Pull request merged**